hadoop-tools/hadoop-azure/dev-support/testrun-scripts/testsupport.sh

```diff
@@ -63,6 +64,28 @@ fnsBlobConfigFileCheck() {
   fi
 }
 
+if ! command -v az &> /dev/null
+then
+  echo "Azure CLI (az) could not be found. Installing Azure CLI..."
+sudo apt update
+sudo apt install -y azure-cli
+echo "Azure CLI installed successfully."
+fi
+
+uploadToAzure() {
+  accountName=$1
+  azureConfigFilePath="${accountSettingsDir}${accountName}${accountConfigFileSuffix}"
+  accountKey=$(xmlstarlet sel -t -v '//property[name = "fs.azure.account.key"]/value' -n $azureConfigFilePath)
+  containerName=$(date +"%Y-%m-%d")
```

Comment:
I would suggest we follow same directory structure as we have in local.
Container name should be `CI Automation Test Logs`
For each job one folder with name `YYYY-MM-DD_HH-MM-SS` containing test log files.

Otherwise, we end up creating a lot of containers which is not needed.

Reply:
Makes sense.
Will create a single container with the year/month/date directory structure